Userland/Libraries/LibWeb/Layout/FormattingContext.h

```diff
@@ -121,6 +121,7 @@ class FormattingContext {
         // Each block in the containing chain adds its own margin and we store the total here.
         CSSPixels left_total_containing_margin;
         CSSPixels right_total_containing_margin;
+        Box const* matching_left_float_box { nullptr };
```

Comment:
No corresponding logic is required for right-side floats?

Reply:
Good question, I don't think it's required since the right floats don't affect the offset. There might be a related bug affecting the available space instead, but I couldn't come up with an example.